recipe/meta.yaml

```diff
@@ -35,7 +33,7 @@ outputs:
         - mdtraj >=1.9.3
         - dask >=2.7.0
         - distributed >=2.7.0
-        - dask-jobqueue >=0.7.0
+        - dask-jobqueue >=0.8.0
```

Comment:
This is an actual change between versions https://github.com/openforcefield/openff-evaluator/pull/467 / https://github.com/openforcefield/openff-evaluator/pull/467/commits/b4a73e6d574cac28d9cf7b0062e4c8218b51d4b0